manifests/t/TimeStored/QStudio/5.01/TimeStored.QStudio.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: TimeStored.QStudio
+PackageVersion: "5.01"
+PackageLocale: en-US
+Publisher: TimeStored
+PackageName: QStudio
+License: Proprietary

```

Comment:
This is probably not correct. The license got [changed](https://www.timestored.com/b/qstudio-now-open-source-release-5-01/) to (open-source) `Apache License 2.0` in 5.01.